bigquery_etl/cli/query.py

```diff
@@ -173,49 +201,63 @@ def create(ctx, name, sql_dir, project_id, owner, dag, no_schedule):
+        no_schedule = True
 
     if dataset.endswith("_derived"):
-        # create directory for this table
-        derived_path = path / project_id / dataset / (name + version)
-        derived_path.mkdir(parents=True)
-
         # create a directory for the corresponding view
-        view_path = path / project_id / dataset.replace("_derived", "") / name
+        create_view_path = True
         # new versions of existing tables may already have a view
-        view_path.mkdir(parents=True, exist_ok=True)
+        view_exist_ok = True
     else:
-        # check if there is a corresponding derived dataset
+        # check if there is a corresponding derived dataset. If so, create
+        # the view path
         if (path / project_id / (dataset + "_derived")).exists():
-            derived_path = path / project_id / (dataset + "_derived") / (name + version)
-            derived_path.mkdir(parents=True)
-            view_path = path / project_id / dataset / name
-            view_path.mkdir(parents=True)
-
             dataset = dataset + "_derived"
-        else:
-            # some dataset that is not specified as _derived
-            # don't automatically create views
-            derived_path = path / project_id / dataset / (name + version)
-            derived_path.mkdir(parents=True)
-
-    click.echo(f"Created query in {derived_path}")
+            create_view_path = True
+
+    table_name = name + version
+    derived_path = path / project_id / dataset / table_name
+    derived_path.mkdir(parents=True)
+    if use_live:
+        use_live_table_name = name + use_live_slug + version
+        use_live_path = path / project_id / dataset / (name + use_live_slug + version)
+        use_live_path.mkdir(parents=True)
+    if create_view_path:
+        view_path = path / project_id / dataset.replace("_derived", "") / name
+        view_path.mkdir(parents=True, exist_ok=view_exist_ok)
 
-    if view_path and not (view_file := view_path / "view.sql").exists():
+    if create_view_path and not (view_file := view_path / "view.sql").exists():
         # Don't overwrite the view_file if it already exists
-        click.echo(f"Created corresponding view in {view_path}")
         view_dataset = dataset.replace("_derived", "")
-        view_file.write_text(
-            reformat(
-                f"""CREATE OR REPLACE VIEW
+
+        if use_live:
+            view_text = f"""CREATE OR REPLACE VIEW
                   `{project_id}.{view_dataset}.{name}`
                 AS SELECT * FROM
-                  `{project_id}.{dataset}.{name}{version}`"""
-            )
-            + "\n"
-        )
+                  `{project_id}.{dataset}.{table_name}`
+                UNION ALL
+                SELECT * FROM
+                  `{project_id}.{dataset}.{use_live_table_name}`
+                WHERE submission_date > (
+                  SELECT MAX(submission_date)
+                  FROM `{project_id}.{dataset}.{table_name}`
+                )"""
```

Comment:
This felt to me like the best way to combine these in the view. This does mean that if the main (daily) table is missing certain days before the most recent, that data will not be filled in by the `_use_live` table, but that feels appropriate -- a missing day in the stable table is a genuine problem in the data; the `_use_live` data is ephemeral and should not be used to paper over missing historical data